.github/workflows/unit_and_integration_tests.yml

```
@@ -0,0 +1,31 @@
+name: Unit & Integration Tests

```

Use something short ... as it is the first info on each commit status and isn't really important info.

```suggestion
name: CI

```